packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
```

Comment:
Has a decision been made on this use case? (date + time picker)

Reply:
About what did we have to make a decision? :)